javascript/chronometer.js

```diff
@@ -1,34 +1,43 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+      this.currentTime++;
+      if (typeof(printTimeCallback) === "function" ) {
+        printTimeCallback();
+      }
+    }, 1000);
+    
   }
 
   getMinutes() {
-    // ... your code goes here
+    let currentTimeMinutes = this.currentTime / 60;
+    return Math.floor(currentTimeMinutes);
   }
 
   getSeconds() {
-    // ... your code goes here
+    let currentTimeSecondsAfterMin = this.currentTime % 60;
+    return currentTimeSecondsAfterMin;
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    return ("0" + value).slice(-2);
   }
 
   stop() {
-    // ... your code goes here
+    clearInterval(this.intervalId);
   }
 
   reset() {
-    // ... your code goes here
+    this.currentTime = 0;
   }
 
   split() {
-    // ... your code goes here
+    return `${this.computeTwoDigitNumber(this.getMinutes())}:${this.computeTwoDigitNumber(this.getSeconds())}`;
```

Comment:
Sometimes this long lines can get messy, it could be a good idead to break this line.

Example:
 ```
  let minutes = this.computeTwoDigitNumber(this.getMinutes())
  let seconds = this.computeTwoDigitNumber(this.getSeconds())

  return `${minutes}:${seconds}`
```